content/en/docs/languages/java/jmx.md

```diff
@@ -0,0 +1,281 @@
+| ------------------------------ | ------------------------------- | ------------------------------------------------ | ------- |
+| `otel.jmx.enabled`             | `OTEL_JMX_ENABLED`              | Enable JMX metric collection                     | `false` |
+| `otel.jmx.target.system`       | `OTEL_JMX_TARGET_SYSTEM`        | Predefined metric set to use                     | none    |
+| `otel.jmx.interval.milliseconds` | `OTEL_JMX_INTERVAL_MILLISECONDS` | Collection interval in milliseconds              | `10000` |
+| `otel.jmx.service.url`         | `OTEL_JMX_SERVICE_URL`          | JMX service URL for remote connections           | none    |
+| `otel.jmx.groovy.script`       | `OTEL_JMX_GROOVY_SCRIPT`        | Path to custom Groovy script for metric mapping  | none    |
+
+### Predefined Target Systems
+
+OpenTelemetry provides predefined metric mappings for popular Java frameworks
+and application servers. Use the `otel.jmx.target.system` property to enable
+them:
+
+**Available target systems:**
+
+- `activemq` - Apache ActiveMQ
+- `cassandra` - Apache Cassandra
+- `hbase` - Apache HBase
+- `hadoop` - Apache Hadoop
+- `jetty` - Eclipse Jetty
+- `kafka` - Apache Kafka
+- `kafka-consumer` - Apache Kafka Consumer
+- `kafka-producer` - Apache Kafka Producer
+- `solr` - Apache Solr
+- `tomcat` - Apache Tomcat
+- `wildfly` - WildFly Application Server
+
+**Example - Monitoring Tomcat:**
+
+```sh
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.jmx.enabled=true \
+  -Dotel.jmx.target.system=tomcat \
+  -jar myapp.jar
+```
+
+**Example - Monitoring Kafka:**
+
+```sh
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.jmx.enabled=true \
+  -Dotel.jmx.target.system=kafka,kafka-producer,kafka-consumer \
+  -jar myapp.jar
+```
+
+{{% alert title="Note" color="info" %}}
+
+You can specify multiple target systems by separating them with commas.
+
+{{% /alert %}}
+
+### Remote JMX Connections
+
+To collect metrics from a remote JVM, configure the JMX service URL:
+
+```sh
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.jmx.enabled=true \
+  -Dotel.jmx.service.url=service:jmx:rmi:///jndi/rmi://localhost:9999/jmxrmi \
+  -jar myapp.jar
+```
+
+The remote JVM must be started with JMX remote enabled:
+
+```sh
+java -Dcom.sun.management.jmxremote \
+  -Dcom.sun.management.jmxremote.port=9999 \
+  -Dcom.sun.management.jmxremote.authenticate=false \
+  -Dcom.sun.management.jmxremote.ssl=false \
+  -jar target-app.jar
+```
+
+{{% alert title="Warning" color="warning" %}}
+
+The example above disables authentication and SSL for simplicity. In production
+environments, always enable authentication and SSL for JMX connections.
+
+{{% /alert %}}
+
+## Custom Metric Mappings
+
+For application-specific MBeans or custom monitoring requirements, you can
+create custom metric mappings using Groovy scripts.
+
+### Creating a Custom Groovy Script
```

Comment:
I would also omit any description of making new groovy scripts for scraping JMX! The [jmx-scraper](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/jmx-scraper) is the new way, and it's configured with yaml instead of groovy. We definitely don't want people reading this to think they should be writing new groovy scripts. 😬 

Most users will use one of the stock/default/out-of-the-box yaml configs, but we could have a section that describes the format of the yaml and how to make a custom one. That could be helpful.